aws_quota/check/elb.py

```diff
@@ -180,32 +188,25 @@ class TargetsPerAZPerNetworkLoadBalancerCountCheck(InstanceQuotaCheck):
     service_code = 'elasticloadbalancing'
```

Comment:
Can we fix the class level comment for this check as well since we determined b) is the right interpretation.